packages/marko-web-inquiry/browser/fields/country.vue

```diff
@@ -784,6 +785,15 @@ export default {
       type: Boolean,
       default: false,
     },
+    lang: {
+      type: String,
```

Comment:
This won't actually respect a non-default value unless you pass the `lang` prop from the parent `default-form` component. :)